frontend/src/pages/workspace/[slug]/1-project-team/setup-team.js

```diff
@@ -24,15 +24,15 @@ const View = ({ psItem }) => {
           with the guidance of the NSC and UNEP
         </p>
       </div>
-      <div className={styles.tableSection}>
-        <h5 className={styles.title}>Team members</h5>
+      <div className="tableSection">
+        <h5 className="title">Team members</h5>
         <SetupTeamTable {...{ psItem, members, setMembers }} />
       </div>
-      <div className={styles.addMemberSection}>
-        <h5 className={styles.title}>Add a New Member</h5>
+      <div className="addMemberSection">
+        <h5 className="title">Add a New Member</h5>
```

Comment:
You should not need a class called "title" since you are already using h5 - which implies title. Then you can target the h5 in the scss